nio/schemas.py

```diff
@@ -1213,6 +1213,32 @@ class Schemas(object):
         ],
     }
 
+    tags = {
+        "type": "object",
+        "properties": {
+            "type": {"type": "string"},
+            "content": {
+                "type": "object",
+                "properties": {
+                    "tags": {"type:": "object"},
+                    "properties": {
```

Comment:
The properties here are in the wrong place, they need to be inside the tags dictionary:
```python
"tags": {
    "type": "object",
    "properties": {
        "order": {...}
    }
}
```

Another thing, since tags can any value you'll have to use `patternProperties` instead of `properties`. Please see some other event schemas that use `patternProperties` (e.g. `key_verification_mac`)